src/components/Signup.js

```diff
@@ -0,0 +1,76 @@
+import React, {useState } from 'react';
+import './Signup.css';
+import PropTypes from 'prop-types';
+
+
+async function signupUser(credentials) {
+    return fetch('https://getinvolvedapi.herokuapp.com/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(credentials)
+    })
+      .then(data => data.json())
+    }
+
+export default function Signup({ setToken }) {
+  const [username, setUserName] = useState();
+  const [password, setPassword] = useState();
+
+  const handleSubmit = async e => {
+    e.preventDefault();
+    fetch('https://getinvolvedapi.herokuapp.com/login').then(response => response.json()).then(async json => {
+      
+      console.log(json);
+      let check = false;
+      let i = 0;
+      for (i=0; i < (json.account_list.length); i ++){
+        console.log(json.account_list[i]);
+        console.log(username);
+        if (json.account_list[i].username === username){
+            if (json.account_list[i].password !== password){
+              check = true;
+            }
+        }
+      }
+      if (check === true){
```

Comment:
What if check is not true? Ie password does not match password for given user? Is there any sort of error handling?

Reply:
it should move on the other condition as intended